onnx/common/ir_pb_converter.cc

```diff
@@ -70,7 +70,7 @@ Tensor tensorProtoToTensor(const ONNX_NAMESPACE::TensorProto & tp) {
     break;
   }
   case ONNX_NAMESPACE::TensorProto_DataType_UNDEFINED:
-    abort();
+    throw std::runtime_error("Unknown tensor data type");
```

Comment:
In other places (e.g. checker and shape inference) we define a custom exception class inherit from std::runtime_error and macros (e.g. `fail_check`, `fail_shape_inference`).